src/microbe_stage/systems/ProcessSystem.cs

```diff
@@ -57,10 +57,15 @@ public ProcessSystem(World world, IParallelRunner runner) : base(world, runner,
     ///   </para>
     /// </remarks>
     public static void ComputeActiveProcessList(IReadOnlyList<IPositionedOrganelle> organelles,
-        [NotNull] ref List<TweakedProcess>? result)
+        [NotNull] ref List<TweakedProcess>? result, Dictionary<BioProcess, float> tempStorage)
     {
         result ??= new List<TweakedProcess>();
 
+        foreach (var process in result)
```

Comment:
Could you add a comment here that this block of code is for remembering the existing speed multiplier states?